recipes/seqscreen/meta.yaml

```diff
@@ -6,11 +6,11 @@ package:
 
 source:
   url: 'https://gitlab.com/treangenlab/seqscreen/-/archive/v{{version}}/seqscreen-v{{version}}.tar.gz'  
-  sha256: e40fd39421527026b13dad1e12acc8c08eb1a4c925c7e0db425322fee616deaf                                  
+  sha256: 310062b35e1ba16131d340988d926ea0cf1cf5cc7b9440ced8f1d42eb6f900f5                                 
```

Comment:
Please bump the upstream version number next time. Those changes are very hard to debug from downstream.